plugins/modules/aci_l3out_eigrp_interface_profile.py

```diff
@@ -0,0 +1,392 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Gaspard Micol (@gmicol) <gmicol@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_l3out_eigrp_interface_profile
+short_description: Manage Layer 3 Outside (L3Out) EIGRP interface profile (eigrp:IfP)
+description:
+- Manage L3Out logical interface profile EIGRP policies on Cisco ACI fabrics.
+options:
+  tenant:
+    description:
+    - The name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  l3out:
+    description:
+    - The name of an existing L3Out.
+    type: str
+    aliases: [ l3out_name ]
+  node_profile:
+    description:
+    - The name of the node profile.
+    type: str
+    aliases: [ node_profile_name, logical_node ]
+  interface_profile:
+    description:
+    - The name of an existing interface profile.
+    type: str
+    aliases: [ name, interface_profile_name, logical_interface ]
```

Comment:
```suggestion
    aliases: [ interface_profile_name, logical_interface ]
```

The `name` was repeated twice, by mistake I commented in the below line.